renv.lock

```diff
@@ -767,16 +767,6 @@
       "Repository": "CRAN",
       "Hash": "9b712b51fabc35d2d737fba3567f1a9b"
     },
-    "mlr3viz": {
```

Comment:
The problem seems to be here. theme_mlr3() function is available in mlr3viz 0.5.10, so bhy removing this part, we have to invoke installing exatly this version. but i docker tries to install 0.6 version which does not have this function which mlr3proba 0.4.0 also depends on

Reply:
I agree i would try another way

Reply:
good